transit-vpc-primary-account-existing-vpc.template

```diff
@@ -80,6 +80,16 @@
       "Type" : "String",
       "Default" : ""
     },
+    "CodeLocationS3Bucket" : {
+      "Description" : "The location of the CISCO code used by this solution",
+      "Type" : "String",
+      "Default" : "solutions-reference",
+    ,
+    "SolutionsHelperS3BucketPrefix" : {
```

Comment:
why have `CodeLocationS3Bucket` specify the full bucket name, but this is a prefix? it might be a bit confusing